internal/handlers/handler_oauth2_revocation.go

```
@@ -3,9 +3,10 @@ package handlers
 import (
 	"net/http"
 
-	oauthelia2 "authelia.com/provider/oauth2"
+	"github.com/authelia/authelia/v4/internal/oidc"
```

golangci-lint (2.11.4)

[error] File is not properly formatted

(goimports)